client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -43,7 +43,7 @@ const EmailProvidersStackedComparison = ( {
 	selectedEmailProviderSlug,
 	selectedIntervalLength = IntervalLength.ANNUALLY,
 	source,
-}: EmailProvidersStackedComparisonProps ): ReactElement => {
+}: EmailProvidersStackedComparisonProps ): JSX.Element => {
```

Comment:
What's the difference returning an empty object `<></> `and returning `null`? In line 76 we are returning empty react element, shall we return null instead? I did this return, but seems to me that is better to return `null` instead and change the return type to be `JSX.Element | null`.

Reply:
So I think that returning `null` is a clearer indication of the component not being renderable. Also, to be completely correct, `<></>` is shorthand for `<React.Fragment></React.Fragment>`, so it's representing an empty React Fragment. I will make this change as well.